sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResultArray.ts

```diff
@@ -5,7 +5,7 @@ import {
   TextDocumentBatchStatistics,
   DocumentError,
   DocumentSentiment,
-  MultiLanguageInput
+  TextDocumentInput
```

Comment:
Just a quick question: renaming in the generated code would mean these changes would get lost with regeneration right?

Reply:
No, it's renamed in the autorest swagger, so any subsequent regeneration will keep the internal rename.